website/common/script/cron.js

```diff
@@ -6,6 +6,7 @@
  */
 import _ from 'lodash';
 import moment from 'moment';
+import 'moment-recur';
```

Comment:
quick question about this: if in the new client we never imports `website/common/script/cron.js` the moment-recur module won't be imported to. Is it necessary to import it for the client side to work? If yes, I'll add it separately